src/main/kotlin/com/example/ramenbm/global/security/SecurityConfig.kt

```diff
@@ -32,6 +32,7 @@ class SecurityConfig(
                     .antMatchers(HttpMethod.POST, "/auth/signup").permitAll()
                     .antMatchers(HttpMethod.POST, "/auth/signin").permitAll()
                     .antMatchers(HttpMethod.PATCH, "/auth/reissue").permitAll()
+                    .antMatchers(HttpMethod.POST, "/api/ramen").permitAll()
```

Comment:
왜 permitAll()로 인증 절차를 제외 시키신건지 궁금합니다!

Reply:
기능에서 admin이나 member 이런 식으로 역할을 나누는 부분이 없기 때문에 permitAll() 을 써서 역할 구분 없이 접근할 수 있도록 했습니다.

Reply:
그러면 hasAnyAuthority(Authority.ROLE_MEMBER.name, Authority.ROLE_ADMIN.name)을 permitAll() 대신에 사용하여 해결 할 수 있을거 같습니다. permitAll()을 사용해버리면 해당 리소스에 대한 모든 접근을 허용해 버려서 인증 된 사용자만 라면 게시글을 작성해야하는데 맞지 않아 보입니다.
```suggestion
                    .antMatchers(HttpMethod.POST, "/api/ramen").hasAnyAuthority(Authority.ROLE_MEMBER.name, Authority.ROLE_ADMIN.name)
```

Reply:
수정했습니다.